maquetado/login.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Log In</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="login-container">
+    <form class="info-container column">
+      <img src ="./assets/wolox.png" class="login-img" alt="Wolox">
+      <h3 class="input-description">Email</h3>
+      <input name="email" id="email" class="input-field">
```

Comment:
Remember to add the type attribute for every input. You have different type like: text, number, email, etc.